data/tests/images.ts

```diff
@@ -96,6 +98,40 @@ const tests: Test[] = [
       },
     },
   },
+  {
+    label: 'false-positives',
+    description:
+      "Due to fuzzy matching on alternative spellings, we need to ensure we aren't too fuzzy.",
+    pass: equalTo0,
+    searchTemplateAugmentation: filterCaseRatings,
+    cases: [
+      { query: 'monsters', ratings: ['n7r5s65w'] }, // shouldn't match "Monastery"
```

Comment:
It'd be nice to have these comments showing up on the UI

Reply:
good idea! happy for me to add a description block to each test case?

Reply:
Sounds good. Probably have it per test, and then allow you to add it to the cases too.